bin/start-shuffle-server.sh

```diff
@@ -93,6 +99,9 @@ JVM_ARGS=" -server \
           -XX:+PrintAdaptiveSizePolicy \
           -XX:+PrintGCDateStamps \
           -XX:+PrintGCTimeStamps \
+          -XX:+CrashOnOutOfMemoryError \
+          -XX:+ExitOnOutOfMemoryError \
+          ${OOM_DUMP_OPTS} \
```

Comment:
I think we can just use `JAVA_OPTS`, because users may need add more options.

Reply:
It all ok, but I think there is "MAX_DIRECT_MEMORY_OPTS", so I added OOM_DUMP_OPTS.

Reply:
I mean we can add a env variable like `spark.driver.extraJavaOptions` which in spark. So that users can add any arguments to `JVM_ARGS` by just add arguments to `JAVA_OPTS` .